packages/web3-validator/src/types.ts

```diff
@@ -5,7 +5,7 @@ export { JSONSchemaType } from 'ajv';
 export { DataValidateFunction, DataValidationCxt } from 'ajv/dist/types';
 
 export type Web3ValidationErrorObject = ErrorObject;
-export type ValidInputTypes = Buffer | bigint | string | number | boolean;
+export type ValidInputTypes = ArrayBuffer | Buffer | bigint | string | number | boolean;
```

Comment:
@nazarhussain This should be okay, right? I added it because it's apart of [Bytes](https://github.com/ChainSafe/web3.js/blob/df60a2dfdd50eaed259d2fd0dc63f2eb3e9f7cf3/packages/web3-utils/src/types.ts#L3)

Reply:
Yes seems good.